Sources/STARLib/FastStrategy.swift

```diff
@@ -14,6 +14,7 @@
 
 import Foundation
 import SwiftSyntax
+import SwiftSyntaxParser
```

Comment:
Why was this import not needed before? Was it just moved into its own package in swift tools 5.6?

Reply:
Initially that was my understanding; however, relooking at the commit history it seems it used to be the case since Swift 5.1. Let me recheck what might have changed or gone wrong

Reply:
Actually this was their package definition for version 5.5:

```swift
let package = Package(
  name: "SwiftSyntax",
  targets: [
    .target(name: "_CSwiftSyntax"),
    .testTarget(name: "SwiftSyntaxTest", dependencies: ["SwiftSyntax"], exclude: ["Inputs"]),
    .target(name: "SwiftSyntaxBuilder", dependencies: ["SwiftSyntax"]),
    .testTarget(name: "SwiftSyntaxBuilderTest", dependencies: ["SwiftSyntaxBuilder"]),
    .target(name: "lit-test-helper", dependencies: ["SwiftSyntax"]),
    .testTarget(name: "PerformanceTest", dependencies: ["SwiftSyntax"])
    // Also see targets added below
  ]
)

package.products.append(.library(name: "SwiftSyntax", type: libraryType, targets: ["SwiftSyntax"]))
package.products.append(.library(name: "SwiftSyntaxBuilder", type: libraryType, targets: ["SwiftSyntaxBuilder"]))
```

This is in Swift 5.6.

```swift
let package = Package(
  name: "SwiftSyntax",
  products: [
    .library(name: "SwiftSyntax", type: .static, targets: ["SwiftSyntax"]),
    .library(name: "SwiftSyntaxParser", type: .static, targets: ["SwiftSyntaxParser"]),
    .library(name: "SwiftSyntaxBuilder", type: .static, targets: ["SwiftSyntaxBuilder"])
  ],
  targets: [.....]
```

As you can see in 5.5 they didn't have a product appended that is `SwiftSyntaxParser`, so that's what changed.

Reply:
Makes sense, thanks for checking